packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -66,16 +65,18 @@ const Dashboard = ({ active_tab, setActiveTab, toggleStrategyModal, is_drawer_op
                     </Tabs>
                 </div>
             </div>
-            <div className='dashboard__run-strategy-wrapper'>
-                <RunStrategy />
-                {/*
+            {!isMobile() && (
```

Comment:
Can't `DesktopWrapper` be used instead of `!isMobile()` here?

Reply:
sure, implemented ✅